src/python/pants/engine/internals/native_engine.pyi

```diff
@@ -662,9 +662,17 @@ class PyOptionId:
         self, *components: str, scope: str | None = None, switch: str | None = None
     ) -> None: ...
 
+class PySplitArgs:
```

Comment:
It's possible that we won't need to access this in Python in the future, but for now I've implemented this, to make the port less invasive, and also so that I can run the existing Python tests against this new implementation.